prow/plugins/approve/approvers/owners.go

```diff
@@ -585,7 +585,7 @@ Associated issue requirement bypassed by: {{range $index, $approval := .ap.ListN
 
 {{ end -}}
 
-The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands).
+The full list of commands accepted by this bot can be found [here](https://github.com/caicloud/engineering/getting-started/bot-commands.md).
```

Comment:
can you update this? i think we've settled down with our doc location.

Reply:
@bbbmj

Reply:
ok, I will make it later